.github/workflows/docker.yml

```diff
@@ -47,6 +47,8 @@ jobs:
         uses: arduino/setup-protoc@v1
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Update aio-max-nr
+        run: echo 1048576 > /proc/sys/fs/aio-max-nr
```

Comment:
Nice!

Reply:
Yeah - this doesn't work on GH runners unfortunately.